state-synchronizer/src/tests/integration_tests.rs

```diff
@@ -194,6 +201,7 @@ impl SynchronizerEnv {
             clients: vec![],
             storage_proxies: vec![],
             signers,
+            network_id_keys,
```

Comment:
hmm... this is a bit unfortunate... lemme know about the tech debt assumption -- otherwise having unauthed could be super useful for testing.

Reply:
I would actually prefer everyone to use as-close-to-prod-as-possible transports, since they will then exercise more realistic code paths.